compiler/pom.xml

```diff
@@ -32,6 +32,16 @@
       <artifactId>auto-value</artifactId>
       <optional>true</optional>
     </dependency>
+    <dependency>
+      <groupId>org.jgrapht</groupId>
```

Comment:
`org.jgrapht.` needs to be relocated (see maven-shade-plugin configuration below)